lib/peep.rb

```diff
@@ -0,0 +1,27 @@
+require 'pg'
+require_relative 'timenow'
+require_relative 'database_connection'
+
+class Peep
+  attr_reader :id, :content, :time
+  def initialize(id:, content:, time:)
+    @id = id
+    @content = content
+    @time = time
+  end
+
+  def self.all
```

Comment:
Instead of what you did there is also an option to call reverse at the end of this method, this would satisfy the user story and keep logic in the model